src/Notes/WooCommercePayments.php

```diff
@@ -98,15 +98,22 @@ public static function possibly_add_note() {
 	 * @return Note
 	 */
 	public static function get_note() {
+		$tos = sprintf(
+			/* translators: 1: opening link tag, 2: closing tag */
+			__( 'By clicking "Get started", you agree to our %1$sTerms of Service%2$s', 'woocommerce-admin' ),
+			'<a href="https://wordpress.com/tos/" target="_blank">',
+			'</a>'
+		);
+
 		$note = new Note();
 		$note->set_title( __( 'Try the new way to get paid', 'woocommerce-admin' ) );
-		$note->set_content( __( 'Securely accept credit and debit cards on your site. Manage transactions without leaving your WordPress dashboard. Only with WooCommerce Payments.', 'woocommerce-admin' ) );
+		$note->set_content( __( 'Securely accept credit and debit cards on your site. Manage transactions without leaving your WordPress dashboard. Only with WooCommerce Payments.', 'woocommerce-admin' ) . '<br><br>' . $tos );
 		$note->set_content_data( (object) array() );
 		$note->set_type( Note::E_WC_ADMIN_NOTE_MARKETING );
 		$note->set_name( self::NOTE_NAME );
 		$note->set_source( 'woocommerce-admin' );
 		$note->add_action( 'learn-more', __( 'Learn more', 'woocommerce-admin' ), 'https://woocommerce.com/payments/', Note::E_WC_ADMIN_NOTE_UNACTIONED );
-		$note->add_action( 'install-now', __( 'Install now', 'woocommerce-admin' ), false, Note::E_WC_ADMIN_NOTE_ACTIONED, true );
+		$note->add_action( 'get-started', __( 'Get started', 'woocommerce-admin' ), wc_admin_url( '&action=install-woocommerce-payments' ), Note::E_WC_ADMIN_NOTE_ACTIONED, true );
```

Comment:
Action name probably should be something different because it installs, activates and sets up the plugin. So `setup-woocommerce-payments` might be better.

Reply:
👍 Done in https://github.com/woocommerce/woocommerce-admin/pull/5413/commits/e049694bd19979e151c5fbf36f6c49dd40dc2fe8